lib/server.js

```diff
@@ -47,8 +48,20 @@ module.exports = (options) => {
         );
       }
     },
-    content: (staticOptions) => {
+    content: staticOptions => {
       middleware.content.called = true;
+
+      const paths = options.content
+        .map(dir => path.relative(process.cwd(), dir))
+        .map(dir => `./${dir}/`);
+
+      if (paths.length === 1) {
+        log.info(chalk`Serving Static Content from: {grey ${paths[0]}}`);
+      } else {
+        log.info('Serving Static Content from:');
+        paths.forEach(path => log.info(chalk.grey(`\t- ${path}`)));
```

Comment:
(not sure why this one didn't make it into the previous review, I blame github)

we tend not to use tabs in console output, since their display from one OS to another can be messy and they tend not to line up consistently. for those reasons, please use spaces here, and please do omit the hyphen. using `log.info` for each path is going to produce the log info prefix for every line, which we also don't want to do here.

The end result should look like:

```console
i ｢serve｣: Serving Static Content from:
            /some/path/first
            /some/path/second
            /some/path/third
```

we also shy away from `forEach` as `for/of` is more performant after Node v6.

Reply:
Oh I didn't know for of was faster good to know 👍
You are right as it is now it is not neatly logging out the paths, i'm on it!